l10n_ch_invoice_reports/models/mail_template.py

```diff
@@ -16,14 +16,10 @@ def generate_email(self, res_ids, fields=None):
 
         result = super().generate_email(res_ids, fields)
         if self._should_print_invoice_attachment():
-            res_ids_to_templates = self.get_email_template(res_ids)
-            for res_id in self.env["account.move"].browse(res_ids):
-                related_record = self.env[self.model_id.model].browse(res_id.id)
-                template = res_ids_to_templates[res_id.id]
-                invoice_attachments = self.generate_invoice_attachment(
-                    template, res_id.id, related_record
-                )
-                result[res_id.id]["attachments"] = invoice_attachments
+            self.get_email_template(res_ids)
```

Comment:
is this call needed here? The result is not used at all

Reply:
@simahawk  my mistake, we don't need it anymore. removed.